WebApi/WebApiTest/ServiceTest/CommentServiceTest.cs

```diff
@@ -0,0 +1,216 @@
+              item.UserID == actualItem.UserID && item.Content == actualItem.Content &&
+              item.PostID == actualItem.PostID)));
+
+
+
+        }
+
+        [Fact]
+        #region TODO
+        public void GetAll_InValidCall()
+        {
+            Assert.True(true);
+
+
+        }
+        #endregion
+        [Fact]
+        #region TODO
+        public void GetLikedUsers_ValidCall()
+        {
+
+            Assert.True(true);
+        }
+        #endregion
+        [Fact]
+        #region TODO
+        public void GetLikedUsers_InValidCall()
+        {
+
+
+            Assert.True(true);
+        }
+        #endregion
+        [Fact]
+       
+        public void AddComment_ValidCall()
+        {
+
+            int initLength = comments.Count();
+            var expected = comments;
+            var newCommentDTO = new CommentDTO() { CommentID = 4, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" };
+            var newComment = new Comment() { CommentID = 4, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" };
+            expected.Add(new Comment { CommentID = 4, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" });
+            var mockICommentRepository = new Mock<ICommentRepository>();
+           
+            mockICommentRepository.Setup(x => x.AddAsync(It.IsAny<Comment>())).Returns(Task.Run(() =>
+            {
+                return new Comment() { CommentID = 4, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" };
+            }));
+
+
+            var commentService = new CommentService(mockICommentRepository.Object);
+            var actual = commentService.AddCommentAsync(UserId, newCommentDTO).Result;
+
+            Assert.True(actual != null);
+            Assert.Equal(newComment.CommentID, actual.CommentID);
+            Assert.Equal(newComment.Content, actual.Content);
+            Assert.Equal(newComment.DateTime, actual.DateTime);
+            Assert.Equal(newComment.PostID, actual.PostID);
+            Assert.Equal(newComment.UserID, actual.UserID);
+            //DLUGOSC
+
+
+
+
+        }
+
+        [Fact]
+        #region TODO
+
+        public void AddComment_InValidCall()
+        {
+            Assert.True(true);
+        }
+        #endregion
+        [Fact]
+        #region TODO
+        public void DeleteComment_ValidCall()
+        {
+
+
+            Assert.True(true);
+
+        }
+        #endregion
+        [Fact]
+        #region TODO
+        public void DeleteComment_InValidCall()
+        {
+            Assert.True(true);
+        }
+        #endregion
+        [Fact]
+
+        public void EditComment_ValidCall()
+        {
+
+
+            var newCommentDTO = new CommentDTO() { CommentID = 2, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" };
+            var newComment = new Comment() { CommentID = 2, UserID = 1, PostID = 1, DateTime = DateTime.Today, Content = "testNowy" };
+
```

Comment:
To samo co wcześniej czyli na sztywno wpisane dane plus DateTime lepiej, żeby nie był zmienny zależnie od dnia ( a tak na prawdę też od serwera który testuje)